CardPayments/src/main/java/com/paypal/android/cardpayments/ApproveOrderListener.kt

```diff
@@ -15,6 +15,9 @@ interface ApproveOrderListener {
     @MainThread
     fun onApproveOrderSuccess(result: CardResult)
```

Comment:
That's interesting. I don't think it's possible to do this in iOS with protocols.
Do you feel like this is something merchant would want us to handle?